src/React/Redux.purs

```diff
@@ -139,9 +135,9 @@ foreign import data REDUX :: Effect
 
 foreign import data Store :: Type -> Type -> Type
 
-foreign import data ReduxReactClass :: Type -> Type -> Type -> Type
+foreign import data ReactClass :: Type -> Type -> Type -> Type
```

Comment:
This will affect error messages.  PureScript doesn't show qualified type names (or maybe it's just a problem of psa?).  This might lead to some confusion when reading an error message.

Reply:
And for this, I think you are looking at an outdated version. This has been renamed to `ConnectClass`.